app/views/courses/index.html.erb

```diff
@@ -14,12 +14,15 @@
 
 <p class="govuk-body">Use this section to:</p>
 
-<ul class="govuk-list govuk-list--bullet">
+<ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-7">
   <li>write about each course</li>
   <li>preview and publish courses</li>
   <li>copy content between courses</li>
 </ul>
 
+<%= link_to "Add a new course", "https://goo.gl/forms/kdNksCuwcz7TVtNF2", class: "govuk-button govuk-!-margin-bottom-2", rel: "noopener noreferrer", target: :blank %>
```

Comment:
We should only show the "Add a new course" link at the top of there's more than 10 courses.

Reply:
Probably also needs to be wrapped in `@provider.opted_in?`?